designer/client/containers/EnvironmentTag.tsx

```diff
@@ -0,0 +1,65 @@
+import {useSelector} from "react-redux"
+import {getFeatureSettings} from "../reducers/selectors/settings"
+import React, {useMemo} from "react"
+import styled from "@emotion/styled"
+
+// TODO: get rid of 'indicator-', maybe rename to "warn", "prod" etc.
+export enum EnvironmentTagColor {
+  green = "indicator-green",
+  red = "indicator-red",
+  blue = "indicator-blue",
+  yellow = "indicator-yellow"
+}
+
+const Tag = styled.div(
+  {
+    display: "inline-block",
+    padding: "3px 8px",
+    marginBottom: 0,
+    fontSize: 15,
+    fontWeight: "normal",
+    textAlign: "center",
+    whiteSpace: "nowrap",
+    verticalAlign: "middle",
+    backgroundImage: "none",
+    borderRadius: "3px",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    letterSpacing: "0.2px",
+    color: "hsl(0,0%,100%)",
+  },
+  ({color}) => ({
+    backgroundColor: color,
+  })
+)
+
+export function EnvironmentTag() {
+  const {environmentAlert: {content, color} = {}} = useSelector(getFeatureSettings)
+  const background = useMemo(
+    () => {
+      switch (color) {
+        case EnvironmentTagColor.green:
```

Comment:
IMO user should provide e.g. "green" instead of "indicator-green". Setting is named "color" so as a user I would expect color, not CSS class